packages/celer-web-app/src/data/storage/RecentPages.ts

```diff
@@ -0,0 +1,41 @@
+import { LocalStorageWrapper } from ".";
+
+const KEY = "RecentlyVisitedPages";
+const NUM_PAGES = 5; // <-- Change this to increase number of pages tracked
+const DEFAULT = new Array(NUM_PAGES);
+
+// Input a string, return an array of max length NUM_PAGES with
+//  that string separated by spaces.
+function parseArrayFromString(valueString: string): string[] {
```

Comment:
Have you tried `LocalStorageWrapper.load<string[]>`? I don't think you need custom parsers here

Reply:
This should be fixed. I tried it on several occasions, but I was getting some typing issues that I couldn't figure out.